jobclient/python/cook/scheduler/client/constraints.py

```diff
@@ -53,18 +53,6 @@ def __init__(self, operator: Operator, attribute: str, value: str):
         self.attribute = attribute
         self.value = value
 
-    def __hash__(self):
-        return hash((self.operator, self.attribute, self.value))
-
-    def __eq__(self, other):
-        if self is other:
-            return True
-        if not isinstance(other, self.__class__):
-            return False
-        return (self.operator == other.operator and
-                self.attribute == other.attribute and
-                self.value == other.value)
-
     def to_list(self) -> list:
         """Generate this constraint's `list` representation."""
         return [
```

Comment:
Do we need any of the constraint stuff in this PR? Seems like maybe we can just rip it all out for now?

Reply:
We probably could and just leave them as raw lists